src/client.rs

```diff
@@ -257,6 +244,18 @@ pub struct ClientStats {
     pub indexes: HashMap<String, IndexStats>,
 }
 
+/// Health of the MeiliSearch server.
+/// Example:
```

Comment:
We usually document like this

```suggestion
/// Health of the MeiliSearch server.
/// 
/// # Example
/// 
```

Reply:
I took the only bad example in the file 😂 
https://github.com/meilisearch/meilisearch-rust/blob/c860dd0af4bd5e8a6ca412026a1a125594cbfd9a/src/client.rs#L267-L275

Reply:
I'll change it this week 🙂